internal/driver/glfw/canvas.go

```diff
@@ -34,7 +35,8 @@ type glCanvas struct {
 
 	scale, detectedScale, texScale float32
 
-	context driver.WithContext
+	context         driver.WithContext
+	webExtraWindows *container.MultipleWindows
```

Comment:
Do we need this field? Is it not possible to always have the web driver have the canvas content be this, even for cases where this is one window?

Reply:
I could not find a way to do that - as the multiple window overlay will take events as per previous discussion.
Bear in mind also that because all the canvas code is common if we want to inject a layer below we would have to override a lot of functionality in that common space to handle the visibility and event handling (like earlier commit on this PR).